python/rikai/spark/sql/model.py

```diff
@@ -155,6 +156,19 @@ def model_type(self) -> "ModelType":
     def load_model(self) -> Any:
         """Load the model artifact specified in this spec"""
 
+    def load_label_fn(self) -> Optional[Callable]:
+        """Load the function that maps label id to human-readable string"""
```

Comment:
should we also add support from SQL parser as well?

Reply:
made #626